src/containers/Tenant/Query/QuerySettingsDialog/constants.ts

```diff
@@ -143,4 +148,9 @@ export const QUERY_SETTINGS_FIELD_SETTINGS = {
     timeout: {
         title: formI18n('form.timeout'),
     },
+
+    // deprecated https://github.com/ydb-platform/ydb-embedded-ui/issues/1239
+    isolationLevel: {
+        title: '',
+    },
```

Comment:
Why do you need it here? Probably, you can remove this field in `useLastQueryExecutionSettings`, then it should not appear anywhere

Reply:
no need